server/config.py

```diff
@@ -0,0 +1,4 @@
+SQLALCHEMY_DATABASE_URI = "postgresql://localhost"
```

Comment:
Is there any particular reason to store this as a `.py` rather than a `.json`, `.yml`, or any other data storage format?

Also I advocate having this be a `config.py.sample` or something that's a template, and we'll just share the correct configuration variables among ourselves.  API keys and stuff can go here.